src/contentscript/contentscript.d.ts.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"contentscript.d.ts","sourceRoot":"","sources":["contentscript.ts"],"names":[],"mappings":";AAmBA,OAAO,sBAAsB,CAAC"}
```

Comment:
What caused the .d.ts.map file to be generated?   I'm not sure it's useful to check in.

Reply:
it came up during first init. Not finding much on the web for its use however...
Will check again